py-polars/tests/test_series.py

```diff
@@ -1127,18 +1128,6 @@ def test_comparisons_bool_series_to_int() -> None:
         srs_bool > 2  # noqa: B015
 
 
-def test_trigonometry_functions() -> None:
```

Comment:
I think these were removed by accident?

Reply:
There is some overlap with these and the `test_trigonometric` tests.

I figured these were no longer required. But there is something to be said for testing the expected values explicitly instead of defering to numpy for this (as in the other test).

What do you think?

Reply:
Oh, now I see the paramaterized test. Jup, seems duplicate.